lib/models/balenaos-contract.ts

```diff
@@ -1,12 +1,12 @@
-import { Contract } from '../types/contract';
+import { Contract } from "../types/contract";
```

Comment:
The quotes would need to be reverted to single ones, otherwise the linter will complain :)

Reply:
Sure, done, thanks for letting me know. Prettify caused this and I later found out the hard way that balena-lint with --fix needs to be used. I've added that line to the README.